data/remnant missions.txt

```diff
@@ -57,8 +55,14 @@ mission "First Contact: Remnant"
 				decline
 	on accept
 		"reputation: Remnant" <?= -10
+		log "Refused to give a blood sample to a group of strange-looking humans that live south of the Core. Upon takeoff, their ships attacked."
+		log "Factions" "Remnant" `A group of unusual-looking humans that live in a secluded section of the galaxy. They are openly hostile to those who refuse to submit blood samples.`
```

Comment:
The last part of this - "They are openly hostile to those who refuse to submit blood samples" - seems a little too specific. It would be like saying, "The Wanderers are hostile toward people who land starships in their gardens": it's probably true, but isn't necessarily one of the faction's most defining characteristics.

It might make more sense to just not get a faction log for the Remnant if you make them hostile.